core/events/domain/tests/test_events.py

```diff
@@ -5,6 +5,7 @@
 from arxiv import taxonomy
 from events import save
 from events.domain import event, agent, submission
+from events.exceptions import InvalidEvent
```

Comment:
Since InvalidEvents are now surfaced rather than being swallowed as before, I changed these tests to evaluate exception conditions rather than the return value of the (now removed) ``valid`` method.